ghcide/src/Development/IDE/LSP/Outline.hs

```diff
@@ -250,10 +251,7 @@ defDocumentSymbol l = DocumentSymbol { .. } where
   _tags           = Nothing
 
 showRdrName :: RdrName -> Text
```

Comment:
inline?

Reply:
I'm just a little worried about whether we're too radical. It turns out we also have some `occNameString` and I keep it for semantic clarity, or you're thinking to rule them all with`printOutputable`?

Reply:
I don't know - I don't know how this code works. I just noticed that it's trivial so maybe it should be inlined.